carbonmark/lib/api.ts

```diff
@@ -167,6 +169,23 @@ export const getVintages = async (): Promise<string[]> => {
   return data;
 };
 
+export const getRetirementCertificate = async (
```

Comment:
@Atmosfearful updated to use the new api for generating the certificate and removed the other unused files for generating certificates in /carbonmark. Let me know if this looks okay.